src/index.js

```diff
@@ -3,5 +3,8 @@
 addEventListener("fetch", (event) => event.respondWith(handleRequest(event)));
 
 async function handleRequest(event) {
+  // Log service version
+  console.log("FASTLY_SERVICE_VERSION:", fastly.env.get('FASTLY_SERVICE_VERSION' || ''));
```

Comment:
```suggestion
  console.log("FASTLY_SERVICE_VERSION:", env.get('FASTLY_SERVICE_VERSION'));
```

Use `env` from the `fastly:env` namespace instead of the global `fastly.`.
You'll need to also insert the following line before the `/// <reference` line above.

```javascript
import { env } from "fastly:env";
```

Reply:
The other thing is, also note the removal of the `|| ''`.   The `|| ''` was misplaced, what you should have written instead was this.

```diff
-fastly.env.get('FASTLY_SERVICE_VERSION' || '')
+fastly.env.get('FASTLY_SERVICE_VERSION') ?? ''
```

However, if you read the [docs for `env`](https://js-compute-reference-docs.edgecompute.app/docs/fastly:env/env), you'll see that it already returns the empty string when if the environment value is not found.  Therefore you don't need this check at all.

Reply:
I tried using `import { env } from "fastly:env";` and `env.get` instead of `fastly.env.get` but it caused an error when I ran `fastly compute serve`:
`Error while running request handler: t.env.get is not a function
Stack:
  bin/index.js/</<@<stdin>:49:63
  bin/index.js/<@<stdin>:50:10`